js/table/change.js

```diff
@@ -155,6 +155,9 @@ function verifyAfterSearchFieldChange (index) {
     var $thisInput = $('input[name=\'criteriaValues[' + index + ']\']');
     // validation for integer type
     if ($thisInput.data('type') === 'INT') {
+        // Trim spaces if it's an integer
+        $thisInput.val($.trim($thisInput.val()));
```

Comment:
```suggestion
        $thisInput.val(trim($thisInput.val()));
```
https://caniuse.com/#feat=mdn-javascript_builtins_string_trim

Reply:
Using the following:

```
$thisInput.val(($thisInput.val()).trim());
```

Reply:
Indeed, thank you ^^
Too much php code in my head ^^